GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_truss.cfg

```diff
@@ -0,0 +1,184 @@
+
+	@RESOURCE[ElectricCharge]
+	{
+		@name = ElectricCharge
+		%amount = 2500
+		%maxAmount = 2500
+	}
+}
+
+//	ISS Triple HRS
+//	Diameter: ?
+//	Length: 22.86m (75ft)
+//	Mass: 3,400kg (7496lb)
+//	Source: http://www.defense-aerospace.com/article-view/release/12155/lockheed-hrs-radiators-on-board-atlantis-(oct.-10).html
+@PART[ht2_radiatorTriple]:FOR[RealismOverhaul]
+{
+	@mass = 3.4
+	@title = Triple HRS radiators
+	@manufacturer = Lockheed Martin
+	@description = The three HRS radiators on S1 weigh approximately 7,500 pounds, which is almost 30 percent of the payload on the Space Shuttle Atlantis.
+	%breakingForce = 200
+	%breakingTorque = 200
+	%fuelCrossFeed = False
+	%radiatorHeadroom = 0.29 // 0.2702 sets the limit to 17C, or 290K
+
+	@MODULE[ModuleActiveRadiator]
+	{
+		@maxEnergyTransfer = 1750
+		@overcoolFactor = 0.0186367
+
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 1.545
+		}
+	}
+}
+
+//	Radial truss port to secure S0 to Destiny module
+@PART[ht2_radialTrussPort]:FOR[RealismOverhaul]
+{
+	@title = S0 radial truss connector
+	@manufacturer = Boeing
+	@description = A radially-attachable mechanism designed for connecting the S0 truss to the ISS.
+}
+
+//	P3/S3 truss
+//	Seperate part in HT2, Originally apart of P4/S4.
+//	Mass: 141.5kg
+@PART[ht2_truss_S2]:FOR[RealismOverhaul]
+{
+	@mass = 0.1415
+	@title = ISS P3/S3 Truss Segment
+	@manufacturer = Boeing
+	@description = The P3/S3 primary structure is made of a hexagonal shaped aluminum structure and includes four bulkheads and six longerons. Used to connect P4/S4, don't forget your dockable truss port!
+}
+
+//	ITS-Z1
+//	Diameter: 4.2m (13.8ft)
+//	Length: 4.9m (16ft)
+//	Mass: 8,755kg (19,300lb)
+//	Source: http://spacecraftearth.com/#smoothscroll-portfolio-module
+//	Source: https://space.skyrocket.de/doc_sdat/its-z1.htm
+@PART[ht2_truss_Z1|ht2_truss_Z1_v2]:FOR[RealismOverhaul]
+{
+	@mass = 8.755
+	@title = ISS Z1 Truss Segment
+	@manufacturer = Boeing
+	@description = The first truss piece, the Z1 truss, launched aboard STS-92 in October 2000. It contains the control moment gyroscope (CMG) assemblies, electrical wiring, communications equipment, and two plasma contactors designed to neutralize the static electrical charge of the space station. Another objective of the Z1 truss was to serve as a temporary mounting position for the "P6 truss and solar array" until its relocation to the end of the P5 truss during STS-120. Though not a part of the main truss, the Z1 truss was the first permanent lattice-work structure for the ISS, very much like a girder, setting the stage for the future addition of the stations major trusses or backbones.
+
+	@MODULE[ModuleReactionWheel]
+	{
+		@PitchTorque = 5
+		@YawTorque = 5
+		@RollTorque = 5
+
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 0.8
+		}
+	}
```

Comment:
TODO: Update <https://github.com/KSP-RO/RealismOverhaul/pull/2629>to account for this as well.